docs/usage.md

```diff
@@ -181,123 +249,236 @@ For example:
 
 Multiple VCF files can be specified if the path must be enclosed in quotes
 
-### --split_fastq
+### --step
 
-Use the Nextflow [`splitFastq`](https://www.nextflow.io/docs/latest/operator.html#splitfastq) operator to specify how many reads should be contained in the split fastq file.
-For example:
+Use this to specify the starting step:
+Default `mapping`
+Available: `mapping`, `prepare_recalibration`, `recalibrate`, `variant_calling`, `annotate`, `Control-FREEC`
 
-```bash
---split_fastq 10000
-```
+### --help
 
-### --sample
+Will display the help message
 
-> :warning: This params is deprecated -- it will be removed in a future release.
-> Please check: [`--input`](#--input)
+### --no_intervals
+
+Disable usage of intervals file, and disable automatic generation of intervals file when none are provided.
+
+### --nucleotides_per_second
 
-Use this to specify the location of your input TSV file, on `mapping`, `recalibrate` and `variantcalling` steps.
+Use this to estimate of how many seconds it will take to call variants on any interval, the default value is `1000` is it's not specified in the `<intervals>.bed` file.
+
+### --sentieon
+
+If [Sentieon](https://www.sentieon.com/) is available, use this to enable it for preprocessing, and variant calling.
+Adds the following tools for the [`--tools`](#--tools) options: `DNAseq`, `DNAscope` and `TNscope`.
+
+More information in the [sentieon](sentieon.md) documentation.
+
+### --skip_qc
+
+Use this to disable specific QC and Reporting tools.
+Available: `all`, `bamQC`, `BaseRecalibrator`, `BCFtools`, `Documentation`, `FastQC`, `MultiQC`, `samtools`, `vcftools`, `versions`
```

Comment:
Can you supply multiple? Or just a single one at a time?

Reply:
Multiple separated by commas

Reply:
I would then state that, beecause at the moment that seems to be the only (exclusive) options, as you have `all` as one option..